cli/pcluster/utils.py

```diff
@@ -754,3 +754,15 @@ def get_cluster_capacity(stack_name):
         if scheduler == "awsbatch"
         else get_asg_settings(stack_name).get("DesiredCapacity")
     )
+
+
+def get_efa_enabled_instance_types():
+    instance_types = []
+
+    for response in paginate_boto3(
```

Comment:
shall we add a try/except to print a specific error message + the message in the ClientError rather than propagating the exception?